### PR TITLE
Instructions for running the demos

### LINKED ISSUE #196: how about some 'getting started' documentation?
I'd like to see the following in a README for this repository:
* Android Studio 2+ instructions
* command line instructions (can I just set some environment variables and run `ndk-build ...`? `gradle ...`? something else?)
* troubleshooting for common problems (for instance, the unspecific error `"A problem occurred starting process 'command 'ndk-build'"`)

Following [these instructions](http://www.gearvrf.org/bin/view/GearVRfDeveloperGuide/GearVRfDevGuide200Start#BuildInstruct) was not enough to get me past this error.

### PR DESCRIPTION
**Pull request by RMKD**
Here's what worked for me on OSX 10.11. Hopefully will help others get started faster. Closes #196

### CODE REVIEW

**Comment by liaxim** [comment]
Looks good

**Comment by liaxim** [comment]
@thomasflynn ? @NolaDonato ?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
 
+## How to run these examples
+* make sure you have [this repository](https://github.com/gearvrf/GearVRf-Demos) and [the GearVR framework](https://github.com/Samsung/GearVRf) in a sibling directory 
+
+```
+git clone https://github.com/Samsung/GearVRf.git
+git clone https://github.com/gearvrf/GearVRf-Demos.git
+```
+
+* if you don't have your computer set up for Android [SDK](https://developer.android.com/studio/index.html) and [NDK](https://developer.android.com/ndk/downloads/index.html) development - the simplest way to get those configured is to use [NVIDIA's Codeworks](https://developer.nvidia.com/codeworks-android) which will automatically configure your `~/.bash_profile`
```
Comment (thomasflynn) [inline README.md]: not familiar with NVIDIA's Codeworks.  looks like you need to sign up to be an NVidia developer in order to download it.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,64 @@
...
+
+* [download the Oculus_Mobile_SDK](https://developer.oculus.com/downloads/mobile/1.0.3/Oculus_Mobile_SDK/) and move the relevant jar files into `GearVRf/GVRf/Framework/framework/src/main/libs/`
```
Comment (thomasflynn) [inline README.md]: the link refers to 1.0.3, which we haven't ported to yet.  and the OVR_MOBILE_SDK variable below is pointing to 1.0.0.1.

**Comment by thomasflynn** [comment]
the oculus version info needs to be consistent and i'm not sure what to do with the Gameworks recommendation.  Everything else looks really good.

**Comment by liaxim** [comment]
Will merge lunch time

**Pull request merged**